src/screens/status/status.vue

```diff
@@ -24,25 +31,51 @@
             <th>version.revision</th>
             <th>version.genesis_KeyBlock_hash</th>
           </tr>
-          <tr>
+          <tr v-if="!isLoading">
             <td><strong>{{ node }}</strong></td>
             <td>{{ nodeStatus.top.keyBlock.height }}</td>
             <td>{{ nodeStatus.top.keyBlock.hash | startAndEnd }}</td>
             <td>{{ nodeStatus.top.keyBlock.time }}</td>
             <td>{{ nodeStatus.version.nodeRevision | startAndEnd }}</td>
             <td>{{ nodeStatus.version.genesisKeyBlockHash | startAndEnd }}</td>
           </tr>
+          <tr v-else>
+            <td><span class="fill-dummy-grey">&nbsp;</span></td>
+            <td><span class="fill-dummy-grey">&nbsp;</span></td>
+            <td><span class="fill-dummy-grey">&nbsp;</span></td>
+            <td><span class="fill-dummy-grey">&nbsp;</span></td>
+            <td><span class="fill-dummy-grey">&nbsp;</span></td>
+            <td><span class="fill-dummy-grey">&nbsp;</span></td>
```

Comment:
Maybe better to render them using `v-for="i in 6"`.

Reply:
@davidyuk that computation is expensive. I don't recommend it

Reply:
On second thought it gives cleaner code so @ror-shubham can you upgrade to Deni's recommendation BUT define the value '6' outside the vue template

Reply:
Loop unrolling it is a too low-level optimization, I think reducing of duplicate markup is more important in this case.
As I understand `6` is the number of columns in the previous row, so maybe not necessary to extract it outside the vue template.